tools/esp8266/app.cpp

```diff
@@ -245,109 +245,101 @@ void app::loop(void) {
                     mPayload[iv->id].requested = true;
 
                     yield();
-                    if(mConfig.serialDebug) {
+                    if (mConfig.serialDebug) {
                         DPRINTLN(DBG_DEBUG, F("app:loop WiFi WiFi.status ") + String(WiFi.status()));
                         DPRINTLN(DBG_INFO, F("Requesting Inverter SN ") + String(iv->serial.u64, HEX));
                     }
-                    if(iv->devControlRequest) {
-                        if(mConfig.serialDebug)
+
+                    if (iv->devControlRequest) {
+                        if (mConfig.serialDebug)
                             DPRINTLN(DBG_INFO, F("Devcontrol request ") + String(iv->devControlCmd) + F(" power limit ") + String(iv->powerLimit[0]));
                         mSys->Radio.sendControlPacket(iv->radioId.u64, iv->devControlCmd, iv->powerLimit);
                         mPayload[iv->id].txCmd = iv->devControlCmd;
                         iv->clearCmdQueue();
                         iv->enqueCommand<InfoCommand>(SystemConfigPara);
-                    }
-                    else {
+                    } else {
```

Comment:
diese Formatierung ist neu für mich, was haltet ihr davon. Ich habe bisher das else in eine neue Zeile geschrieben

Reply:
Ich kannte es schon immer so. :) 
Hier auch Beispiele: https://www.c-howto.de/tutorial/verzweigungen/if-und-else/